src/consumers.ts

```diff
@@ -31,6 +31,8 @@ interface IConsumer
 {
     binding: IBindingDescriptor;
     item: any;
+    vulnerabilityCount: number;
+    advisoryCount: number;
```

Comment:
This is not required, as explained use typecast to access SecurityEngine properties.
```suggestion
```